Project.toml

```
@@ -98,9 +98,9 @@ MacroTools = "0.5.13"
 Markdown = "1.6"
 Ncurses_jll = "6.4.1"
 Pidfile = "1.3"
-Pkg = "1.6"
 REPL = "1.6"
 Random = "1.6"
+SHA = "<0.0.1, 1"
```

```suggestion
SHA = "<0.0.1, 0.7, 1"
```
this was available as v0.7 in some julia versions. Don't ask why, but this should fix the failures